noetic/distribution.yaml

```diff
@@ -407,6 +407,16 @@ repositories:
       url: https://github.com/OTL/cv_camera.git
       version: master
     status: maintained
+  dataspeed_ulc_ros:
+    release:
```

Comment:
In Melodic, you had a doc and source entry for this package.  It isn't required to merge this for Noetic, but it is nicer to the users.  Would you like to add those entries?

Reply:
Sorry, that was an oversight on my part! I just update the PR and it should be good now. Thanks for pointing it out.